### DISCUSSION TITLE
Isochrone Reaching Inaccessible Areas

### DISCUSSION THREAD

**Discussion by pipo3944** [valhalla/valhalla#4748] [Q&A] [answered]
Hello,
I'm using Valhalla to calculate isochrones for some areas in Japan, based on the Japan OpenStreetMap data (japan-latest.osm.pbf).
I encountered an issue where isochrones include inaccessible areas like rivers.

For example:  

**Settings:**
- **Starting Point:** lat: 35.6802, lon: 139.7585
- **Travel Time:** 10 min
- **Mode of Transportation:** auto (= car)

**Request URL:**
```
http://localhost:8082/isochrone?json={"locations":[{"lat":35.67725,"lon":139.7743166}],"costing":"auto
```

![6b1d5fd0-b644-4010-8e21-597d3b508fa5](https://github.com/gis-ops/docker-valhalla/assets/12875256/9899075b-9464-4252-8cfa-c7a0eb9cbd9a)
*[10-minute car isochrone result from Valhalla (purple), OTP (red)]*

Comparing car isochrones from Valhalla (purple) and OpenTripPlanner (red) from the same location (red point):
- Valhalla's isochrone includes the river area (arrow ①), which should be inaccessible.
- Valhalla's isochrone includes the Imperial Palace (arrow ②), which should also be inaccessible to the public. OpenTripPlanner correctly excludes it.

However, Valhalla correctly recognizes the Imperial Palace as inaccessible for pedestrians, as seen in the 40-minute walking isochrone (picture below).

![41f3b2f5-55d0-47f7-867b-80f47e20572f](https://github.com/gis-ops/docker-valhalla/assets/12875256/36fcc163-d5c1-4215-bcd1-cd34f776ffba)

Are there any ways to correctly exclude such inherently inaccessible areas from the isochrone results?
Thank you for your assistance.

**Comment by nilsnolde** [comment, marked as answer]
you're just falling victim to our default params. try `denoise=0` and you'll see tons of holes. you can play around with that. another issue is that ped isochrones are way too coarse so it'll include unwanted areas simply by the coarse resolution of the underlying rasterization process. there's an open ticket for that somewhere.

**Reply by pipo3944** [reply]
@nilsnolde 
Thank you for the prompt response!

I overlooked the option in the API request (I apologize for that).
https://valhalla.github.io/valhalla/api/isochrone/api-reference/#other-request-parameters

After setting denoise to 0, as suggested, I was able to obtain the expected isochrone results!
![スクリーンショット 2024-06-11 11 06 00](https://github.com/valhalla/valhalla/assets/12875256/16a436fa-1f3e-4d73-a657-c48ec3aeec8a)


I'm sorry for the delayed reply. I appreciate your continuous support and will continue to make use of Valhalla in our project.

Thank you again, and best regards!
